docs/tuning.md

```diff
@@ -264,6 +264,13 @@ parent RDD's number of partitions. You can pass the level of parallelism as a se
 or set the config property `spark.default.parallelism` to change the default.
 In general, we recommend 2-3 tasks per CPU core in your cluster.
 
+Sometimes you may also need to increase directory listing parallelism when job input has large number of directories,
```

Comment:
How about making a new subsection (e.g., `## XXXX`) for this?

Reply:
sure will do

Reply:
Yep. +1 for @maropu 's suggestion. This content looks worthy for that.

Reply:
👍